media/linux/pds-sqlite3-queries/create-training-rosters.py

```diff
@@ -52,13 +63,36 @@ def pretty_member(member):
             val = '{ph} {type}'.format(ph=phone['number'], type=phone['type'])
```

Comment:
Should we make this "find good phone numbers" functionality common (i.e., put it in `PDSChurch.py`)?  It seems like a variation of it is implemented in several `.py` files, which means we should probably make it common code (vs. implementing it multiple times).